app/models/card.rb

```diff
@@ -4,20 +4,32 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, :review_date,
             presence: { message: 'Необходимо заполнить поле.' }
 
-  protected
-    def set_review_date
-      self.review_date = Time.now + 3.days
+  def self.pending(id)
+    if id
+      where('review_date <= ? AND id = ?', Time.now, id).order('RANDOM()')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.